Doc/library/inspect.rst

```diff
@@ -215,6 +215,10 @@ attributes:
 +-----------+-------------------+---------------------------+
 |           | cr_code           | code                      |
 +-----------+-------------------+---------------------------+
+|           | cr_origin         | where coroutine was       |
+|           |                   | created, if coroutine     |
+|           |                   | origin tracking is enabled|
++-----------+-------------------+---------------------------+
```

Comment:
Can you add a link to the `set_coroutine_origin_tracking_depth` documentation snippet?

Reply:
Well, that was my original thought. The problem is that
```
:func:`set_coroutine_origin_tracking_depth`
```
is too long to fit in the ascii-art box. So... either we need a shorter name for the function, or we need to redraw this whole giant table, and I couldn't think of a satisfactory way to do either in the 2 minutes I spent thinking about it :-). Any suggestions?

I guess we could use that weird ReST substitution thing? I'm not sure how that works.

Reply:
Does this work: http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets ?